bootstrap-joining-demo/kubernetes-api/kubernetes/akka-cluster.yml

```diff
@@ -70,3 +70,26 @@ spec:
   - protocol: TCP
     port: 8558
     targetPort: 8558
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
So sneaking this in right away I guess; sure we're ok with that? (I think we are but double checking)

Reply:
I don't *really* know, I just copy-pasted the recommendation from the docs into the example project